features/dd-sdk-android-session-replay/src/main/kotlin/com/datadog/android/sessionreplay/internal/recorder/base64/Base64Serializer.kt

```diff
@@ -169,25 +169,31 @@ internal class Base64Serializer private constructor(
         imageWireframe: MobileSegment.Wireframe.ImageWireframe,
         base64SerializerCallback: Base64SerializerCallback?
     ): Bitmap? {
-        var result: Bitmap? = null
         if (shouldUseDrawableBitmap(drawable)) {
-            drawableUtils.createScaledBitmap(
-                (drawable as BitmapDrawable).bitmap
-            )?.let { scaledBitmap ->
-                val shouldCacheBitmap = scaledBitmap != drawable.bitmap
-
-                serializeBitmapAsynchronously(
-                    drawable,
-                    scaledBitmap,
-                    shouldCacheBitmap,
-                    imageWireframe,
-                    base64SerializerCallback
-                )
-
-                result = scaledBitmap
-            }
+            val bitmap = (drawable as BitmapDrawable).bitmap
```

Comment:
Is it a safe cast? Maybe we should change a signature of `shouldUseDrawableBitmap` method to accept only `BitmapDrawable`? This has the following benefits: it helps compiler to use smart-cast, no cast is needed; also the name `shouldUseDrawableBitmap` gets more sense, because it accepts only `BitmapDrawable`